src/utils/businessHours.test.ts

```diff
@@ -279,7 +279,9 @@ describe('businessHours tests', function () {
     });
 
     it('should calculate SLO violation if label is unrouted', async function () {
-      const result = await calculateSLOViolationRoute(UNROUTED_LABEL);
+      const result = await calculateSLOViolationRoute(
```

Comment:
nit: Not strictly in this PR, but still confusing: on L262, I found it pretty hard to parse the double negative "not unrouted". Maybe just "if label is routed"?